app/src/main/java/com/github/palFinderTeam/palfinder/meetups/fragments/CriterionsFragment.kt

```diff
@@ -72,21 +73,36 @@ class CriterionsFragment(val viewModel: MeetUpCreationViewModel) : DialogFragmen
                 getString(R.string.radio_male_and_female) -> viewModel.setCriterionGender(CriterionGender.ALL)
             }
 
-            if (rangeSeekBar.selectedMaxValue == rangeSeekBar.absoluteMaxValue) {
-                viewModel.setCriterionAge(Pair(rangeSeekBar.selectedMinValue, Int.MAX_VALUE))
+            // 66+ means anything above
+            if (radiusSlider.values[1] == radiusSlider.valueTo) {
+                viewModel.setCriterionAge(Pair(radiusSlider.values[0].toInt(), Int.MAX_VALUE))

```

Comment:
Why max value and not MAX_AGE or 0 or -1 ?

Reply:
@atappy decided to use max value for this, I think both are fine